ext/html/html.go

```diff
@@ -3,14 +3,15 @@ package goproxy_html
 
 import (
 	"bytes"
-	"code.google.com/p/go-charset/charset"
-	_ "code.google.com/p/go-charset/data"
 	"errors"
-	"github.com/elazarl/goproxy"
 	"io"
 	"io/ioutil"
 	"net/http"
 	"strings"
+
+	"github.com/datasift/go-charset/charset"
```

Comment:
Do we actually need to change the import name here? If we're keeping the same SHA as before, shouldn't the old import path (but with the `source` directive) still work?